aries-site/src/examples/templates/forms/do&dont/ColumnFormDont.js

```diff
@@ -0,0 +1,355 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  MaskedInput,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+
+const emailMask = [
+  {
+    regexp: /^[\w\-_.]+$/,
+    placeholder: 'jane.smith',
+  },
+  { fixed: '@' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'hpe',
+  },
+  { fixed: '.' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'com',
+  },
+];
+
+const states = [
+  'AK',
+  'AL',
+  'AR',
+  'AZ',
+  'CA',
+  'CO',
+  'CT',
+  'DC',
+  'DE',
+  'FL',
+  'GA',
+  'HI',
+  'IA',
+  'ID',
+  'IL',
+  'IN',
+  'KS',
+  'KY',
+  'LA',
+  'MA',
+  'ME',
+  'MD',
+  'MN',
+  'MO',
+  'MS',
+  'MT',
+  'NC',
+  'ND',
+  'NE',
+  'NH',
+  'NJ',
+  'NM',
+  'NY',
+  'NV',
+  'OH',
+  'OK',
+  'OR',
+  'PA',
+  'RI',
+  'SC',
+  'SD',
+  'TN',
+  'TX',
+  'UT',
+  'VA',
+  'VT',
+  'WA',
+  'WI',
+  'WV',
+  'WY',
+];
+
+const phoneMask = [
+  { fixed: '(' },
+  {
+    length: 3,
+    regexp: /^[0-9]{1,3}$/,
+    placeholder: 'XXX',
+  },
+  { fixed: ')' },
+  { fixed: ' ' },
+  {
+    length: 3,
+    regexp: /^[0-9]{1,3}$/,
+    placeholder: 'XXX',
+  },
+  { fixed: '-' },
+  {
+    length: 4,
+    regexp: /^[0-9]{1,4}$/,
+    placeholder: 'XXXX',
+  },
+];
+
+const emailValidation = [
```

Comment:
Can we extract these email and other validations into a reusable util file? feels like we use these in a few forms so would be nice to pull from one place.

Reply:
done